src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
Babel 5.0 is out with some [export extensions](http://babeljs.io/blog/2015/03/31/5.0.0/#stage-1:-export-extensions) including `export * as ns from "mod";`. I can't tell yet if that helps this util exporting discussion.